manifests/params.pp

```diff
@@ -19,6 +19,8 @@
 # Sample Usage:
 #
 class apache::params inherits ::apache::version {
+  include apache::globals
```

Comment:
Looking at this it might be better to change ```apache::version``` to accept parameters rather than to introduce ```apache::globals```.